src/SDKs/Batch/Management/Management.Batch.Tests/ScenarioTests/CertificateTests.ScenarioTests.cs

```diff
@@ -0,0 +1,104 @@
+                    param.Password = "nodesdk";
+                    certResponse = await this.BatchManagementClient.Certificate.UpdateAsync(resourceGroupName, batchAccountName, batchCertName, param);
+                    Assert.Equal(referenceId, certResponse.Id);
+
+                    // Cancel the deletetion
+                    await this.BatchManagementClient.Certificate.CancelDeletionAsync(resourceGroupName, batchAccountName, batchCertName);
+
+                    // Delete the certificate
+                    try
+                    {
+                        await this.BatchManagementClient.Certificate.DeleteAsync(resourceGroupName, batchAccountName, batchCertName);
+                    }
+                    catch (CloudException ex)
+                    {
+                        if (ex.Response.StatusCode != HttpStatusCode.NotFound)
+                        {
+                            throw;
+                        }
+                    }
+
+                    // Verify certificate was deleted. A GET operation will return a 404 error and result in an exception
+                    try
+                    {
+                        await this.BatchManagementClient.Certificate.GetAsync(resourceGroupName, batchAccountName, batchCertName);
+                    }
+                    catch (CloudException ex)
+                    {
+                        Assert.Equal(HttpStatusCode.NotFound, ex.Response.StatusCode);
+                    }
+
+                }
+                finally
+                {
+                    // Clean up
+                    await this.BatchManagementClient.BatchAccount.DeleteAsync(resourceGroupName, batchAccountName);
```

Comment:
You can just skip this since you're deleting the RG right?

Reply:
Yes. It is OK to delete account too, to align with the creation. Won't waste time.